### PR TITLE
remove testrunner.py

### PR DESCRIPTION
**Pull request by nickpell**
https://github.com/dimagi/commcare-hq/pull/10639#issuecomment-225516234

### CODE REVIEW

**Comment by dimagimon** [comment]
Could not review pull request. It may be too large, or contain no reviewable changes.

**Comment by snopoke** [comment]
:+1: on :white_check_mark:

**Comment by nickpell** [comment]
✅ 
@esoergel @millerdev

File: testrunner.py [hunk 1]
```diff
@@ -1,462 +0,0 @@
...
-        class_name = 'GroupTestRunner{}'.format(i)
-        globals()[class_name] = type(
-            class_name,
-            (_OnlySpecificApps, DbOnlyTestRunner),
-            {
-                'app_labels': settings.TRAVIS_TEST_GROUPS[i]
-            }
-        )
-
-_bootstrap_group_test_runners()
```
Comment (esoergel) [inline testrunner.py]: :boom: :hammer:

**Pull request merged**